PanCardView/CoverFlowView.cs

```diff
@@ -56,12 +56,86 @@ public class CoverFlowView : AbsoluteLayout
         /// </summary>
         public static readonly BindableProperty FirstItemPositionProperty = BindableProperty.Create(nameof(FirstItemPosition), typeof(CoverItemPosition), typeof(CoverFlowView), CoverItemPosition.Left);
 
-
         /// <summary>
         /// The view position property.
         /// </summary>
-        public static readonly BindableProperty ViewPositionProperty = BindableProperty.Create(nameof(ViewPosition), typeof(CoverItemPosition), typeof(CoverFlowView), CoverItemPosition.Center);
+        public static readonly BindableProperty ViewAlignmentProperty = BindableProperty.Create(nameof(ViewAlignment), typeof(CoverItemPosition), typeof(CoverFlowView), CoverItemPosition.Center);
+
+        private readonly PanGestureRecognizer _panGesture = new PanGestureRecognizer();
+        private bool IsViewsInited = false;
+        private bool _isOrientationChanged = false;
+
+        private double _width = -1;
+        private double _height = -1;
+        private double _tmpTotalX = 0;
+
+        private List<View> _recycledViews = new List<View>();
+        private BaseCoverFlowProcessor ViewProcessor { get; }
+
+        public CoverFlowView(BaseCoverFlowProcessor baseCoverFlowProcessor)
+        {
+            this.ViewProcessor = baseCoverFlowProcessor;
+            this.ViewProcessor.CoverFlow = this;
+            this.IsClippedToBounds = true;
+
+            SetPanGesture();
+        }
+
+        public CoverFlowView()
+        {
+            this.ViewProcessor = new BaseCoverFlowProcessor(this);
+            this.IsClippedToBounds = true;
+
+            SetPanGesture();
+        }
+
+        /// <summary>
+        /// Gets or set the displayed views list.
+        /// </summary>
+        /// <value>The displayed views list.</value>
+        private IAbsoluteList<View> DisplayedViews => Children;
```

Comment:
Redundant property. Why can't we work with Children directly ?)

Reply:
Yeah it's redundant but more explicit,
I will keep it for my personal comfort when reading the code.

Anyway Feel free to remove it, if it's annoying you.